packages/components/time-picker/src/time-picker-com/basic-time-spinner.vue

```diff
@@ -4,7 +4,7 @@
       <el-scrollbar
         v-for="item in spinnerItems"
         :key="item"
-        :ref="getRefId(item)"
+        :ref="(scollbar) => setRef(scollbar, item)"
```

Comment:
**Nitpick:** can we move this function in setup so that we can have a cleaner template?